api/server/router/image/image_routes.go

```diff
@@ -537,3 +550,13 @@ func validateRepoName(name reference.Named) error {
 	}
 	return nil
 }
+
+func parsePlatform(platformStr string) (ocispec.Platform, error) {
+	switch platformStr {
+	case "remote":
+		return platforms.DefaultSpec(), nil
+	case "local":
```

Comment:
Do we really want the daemon to know about the special, client-only `local` value for platforms?

Reply:
Not really needed, but wanted to give a meaningful error in case someone uses it via the API client.

Actually, instead of handling `local` in the CLI, maybe a better approach would be to handle it in the `client` package directly?

Reply:
Nice 👍